cdk_infra/lib/cluster-deployment.ts

```diff
@@ -57,7 +56,8 @@ export function deployClusters(
         name: ec2Cluster.name,
         vpc: vpc,
         version: versionKubernetes,
-        instance_type: ec2Cluster.instance_type,
+        instanceTypes: [new InstanceType(ec2Cluster.instance_type.toLowerCase())],
+        amiType: ec2Cluster.name.match('\-arm64\-') ? NodegroupAmiType.AL2_ARM_64 : NodegroupAmiType.AL2_X86_64,
```

Comment:
nit: We could pass the AMI type from the test_cluster.yaml file as well, so that we don't need to rely on the name.

https://github.com/aws/aws-cdk/blob/v1.189.0/packages/@aws-cdk/aws-eks/lib/managed-nodegroup.ts#L24

Reply:
We could, but we already had the convention of including the architecture in the cluster name and matching on that is likely to be less error-prone than correctly copying the magic AMI type string.  This also ensures that we only use one of these two AMI types.